frontend/src/pages/Dashboard/DashboardPage.tsx

```diff
@@ -21,20 +23,25 @@ interface Topic {
 const DashboardPage = () => {
   const topics = useLoaderData() as Topic[];
 
+  const navItems = [<HomePage topics={topics} />, <div>2 </div>, <div>3 </div>];
+  const navHeadinds = [
+    <>
+      <GridViewOutlinedIcon />
+      <Typography variant="overline">Home</Typography>
+    </>,
+    <>
+      <LightbulbOutlinedIcon />
+      <Typography variant="overline">Learn</Typography>
+    </>,
+    <>
+      <SettingsOutlinedIcon />
+      <Typography variant="overline">Settings</Typography>
+    </>,
+  ];
```

Comment:
The reason I have these arrays, is because I'm thinking the Sidenav component should be reusable later. Could potentially use it in the code editor to achieve this functionality of having a 'notes' section with the code editor 
<img width="564" alt="image" src="https://github.com/UOA-AMEA020-SOFTENG-Projects/technical-interview-webapp/assets/62448681/014dd16f-bc4e-49fa-b193-c0fc51dc8abd">

Reply:
yeah makes sense to me